### PR TITLE
Adds basic NFT support

### PR DESCRIPTION
**Pull request by hex337**
- Added NFT model
- Added new commands: `create nft`, `transfer nft`, `my nfts`, `all nfts`
- Did a refactor on how we handle responses to be able to do rich
  responses later via a `BotResponse` struct that supports message
  attachements

### CODE REVIEW

**Review by matt-taylor — APPROVED** [review]
As an expert in Go(🚫) , 🚢  it!

File: cmd/migration/main.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ func main() {
 	config.DBOpen()
 
 	// TODO SELECT pg_try_advisory_lock(migration); and if f don't run migration as they are already happening
-	config.DB.AutoMigrate(&model.User{}, &model.Coin{}, &model.Transaction{})
+	config.DB.AutoMigrate(&model.User{}, &model.Coin{}, &model.Transaction{}, &model.Nft{})
```
Comment (djechlin) [inline cmd/migration/main.go:20]: Pass in the whole model?

File: command/all_nfts.go [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hex337/alex-koin-go/config"
+	"github.com/hex337/alex-koin-go/model"
+)
+
+type AllNftsCommand struct{}
```
Comment (alessandrochime) [inline command/all_nfts.go:11]: you can create type functions instead of empty structs

File: cmd/migration/main.go (hunk 1 above)
Comment (hex337) [reply cmd/migration/main.go:20]: What do you mean?

File: command/all_nfts.go (hunk 2 above)
Comment (hex337) [reply command/all_nfts.go:11]: Can or should? What's the difference and can you write up an example? Maybe do that as part of a separate PR?

**Pull request merged**